plugins/module_utils/network/vyos/argspec/firewall_rules/firewall_rules.py

```diff
@@ -159,22 +169,23 @@ def __init__(self, **kwargs):
                                 },
                                 "inbound_interface": {
                                     "options": {
-                                        "group": {
-                                            "type": "str",
-                                        },
-                                        "name": {
-                                            "type": "str",
-                                        },
+                                        "group": {"type": "str"},
+                                        "name": {"type": "str"},
                                     },
                                     "type": "dict",
                                 },
                                 "ipsec": {
-                                    "choices": ["match-ipsec", "match-none", "match-ipsec-in", "match-ipsec-out", "match-none-in", "match-none-out"],
-                                    "type": "str"
-                                },
-                                "jump_target": {
-                                    "type": "str"
+                                    "choices": [
+                                        "match-ipsec",
+                                        "match-none",
+                                        "match-ipsec-in",
+                                        "match-ipsec-out",
+                                        "match-none-in",
+                                        "match-none-out",
```

Comment:
There are a lot of choices. Are they correct?
```
vyos@r15# set firewall ipv4 input filter rule 100 ipsec 
Possible completions:
   match-ipsec          Inbound IPsec packets
   match-none           Inbound non-IPsec packets


```

Reply:
These have to support all versions that we support, which for this module is 1.3-1.5. 
```
vyos@vv15.dc.cluetrust.com# set firewall ipv4 input filter rule 100 ipsec 
Possible completions:
   match-ipsec-in       Inbound traffic that was IPsec encapsulated
   match-none-in        Inbound traffic that was not IPsec encapsulated
vyos@vv15.dc.cluetrust.com# set firewall ipv4 output  filter rule 100 ipsec 
Possible completions:
   match-ipsec-out      Outbound traffic to be IPsec encapsulated
   match-none-out       Outbound traffic that will not be IPsec encapsulated
vyos@vv15.dc.cluetrust.com# set firewall ipv4 forward filter rule 100 ipsec 
Possible completions:
   match-ipsec-in       Inbound traffic that was IPsec encapsulated
   match-ipsec-out      Outbound traffic to be IPsec encapsulated
   match-none-in        Inbound traffic that was not IPsec encapsulated
   match-none-out       Outbound traffic that will not be IPsec encapsulated
vyos@vv15.dc.cluetrust.com:~$ show version 
Version:          VyOS 1.5-rolling-202411070006
Release train:    current
```